Makefile

```diff
@@ -0,0 +1,60 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the MIT License. See the LICENSE accompanying this file
+# for the specific language governing permissions and limitations under
+# the License.
+
+pkgname=amazon-ec2-net-utils
+version=1.5
+
+prefix=/usr/local
+sysconfdir=${DESTDIR}/etc
+sbindir=${DESTDIR}/${prefix}/sbin
+systemddir=${DESTDIR}/${prefix}/lib/systemd
+udevdir=${DESTDIR}/${prefix}/lib/udev
+mandir=${DESTDIR}/${prefix}/share/man
+
+# We support installation of some init-system dependent files, and
+# default to systemd.
+init=systemd
+
+define install-file
+test -d $2 || install -d -o root -g root -m755 $2
+install -o root -g root -m644 $1 $2
+endef
+
+define install-exe
+test -d $2 || install -d -o root -g root -m755 $2
+install -o root -g root -m755 $1 $2
+endef
+
+sources:
+	git archive --prefix ${pkgname}-${version}/ HEAD | gzip --best > ../${pkgname}-${version}.tar.gz
```

Comment:
you should be able to do this with the invocation to `git-archive`, unless that's missing in some old `git` we have?

Reply:
Nope, just old habits of mine. `git-archive` has probably had this functionality for years, but I'll double check.

Reply:
You can also pick a different name for the archive and use `%setup -n` if you want to avoid `${pkgname}-${version}.tar.gz`.